chapters/install/os/jina-check.md

```diff
@@ -0,0 +1,12 @@
+
+# Check The Installation
+
+After you have installed Jina Core, it is advisable to carry out a short check. This will verify that everything has gone smoothly. On the command line, run the following command:
+
+```bash
+jina -vf
+```
+
+The option `-vf` (long version: `--version-full`) shows Jina and all the dependencies' versions. The output contains a list of Python packages followed by the names of variables that are in use. The figure below shows you how the output looks like.
+
+![jina-version-full](jina-verify.png "Show Jina and its dependencies")
```

Comment:
@nan-wang this is a problem because the output will be quickly outdated. Any suggestion what we should do?

Reply:
Good point! I'd suggest to remove the figure showing outputs. It is definitely ok because the purpose is to check the installation. If jina is not properly installed, one will end up with errors when running `jina -vf` . @hofmannedv